vimdoc/error.py

```diff
@@ -116,6 +116,18 @@ def __init__(self, section):
         'Section {} never defined.'.format(section))
 
 
+class DuplicateSection(BadStructure):
+  def __init__(self, section):
+    super(DuplicateSection, self).__init__(
```

Comment:
nit: since (IIRC) Vimdoc now requires Python 3, this can just be `super().__init__(`.